src/utils/handleError.ts

```diff
@@ -17,7 +17,11 @@ const handleError = (log: Logger, configName: string, req: Request, res: Respons
       res.status(400).json(API_RESPONSE(null, error.name, error.message)).end()
     }
   } else {
-    res.status(500).json(API_RESPONSE(null, null, API_UNEXPECTED_EXCEPTION(`${req.method} ${configName}`))).end()
+    if (error !== null && error.message === 'Decrypting provisioning certificate failed') {
```

Comment:
The handle error file is meant to be more of a generic handler, I don't want to put specific error messages here.

Reply:
Resolved